new_app_setup.py

```diff
@@ -104,163 +87,114 @@ def create_kubernetes_namespace_yml(namespace_filename, team_name):
-    msg['Subject'] = email_contents_subject
-
-    msg.attach(MIMEText(email_contents_body, 'plain'))
-
-    attachment = open(config_filename, "rb")
-
-    # Allows us to upload the attachment
-    part = MIMEBase('application', 'octet-stream')
-    part.set_payload((attachment).read())
-    encoders.encode_base64(part)
-    part.add_header('Content-Disposition', "attachment; filename= " +
-                    config_filename)
-
-    msg.attach(part)  # for the attachment
-    text = msg.as_string()
-    server = smtplib.SMTP('smtp.gmail.com', 587)
-    server.starttls()
-    server.login(MVP_email_address, MVP_email_address_password)
-
-    if type(email_to) is list:
-        for user in email_to.split(','):
-            receiver_email_address = user
-            msg['To'] = receiver_email_address
-            server.sendmail(MVP_email_address, receiver_email_address, text)
-            print ("The config file for {} has successfully been sent "
-                   "to {}".format(team_name, user))
-
-    else:
-        receiver_email_address = email_to
-        msg['To'] = receiver_email_address
-        server.sendmail(MVP_email_address, receiver_email_address, text)
-        print ("The config file for {} has successfully been sent "
-               "to {}".format(team_name, email_to))
-
-    server.close()
-
-
-def add_users(filename, group_user, list_of_users):
-    '''
-    QUESTIONS: I am still unclear after looking into this a good amount how we
-    are specifically wanting users added. I don't know how to have each user be
-    part of a specific namespace/change their permissions. I read a bit about
-    creating a policy file, but I am not sure if that is the route we were
-    intending.
-    '''
-    subprocess.check_call(['kubectl', 'create', 'sa', group_user])
-
-    print("A Kubernetes service account for {} has been created under "
-          "{}".format(group_user, team_name))
-
-    # QUESTION: Did we still want to add additional users for the group
-    # if they provide us a list of them?
-    for user in list_of_users.split(','):
-        subprocess.check_call(['kubectl', 'create', 'sa', user])
-        print("A Kubernetes service account for {} has been created "
-              "under {}".format(user, team_name))
+def get_access_token(config_file, team_name):
+    """
+    This function is using the configuration file we created with
+    get_congif_yml_file and the parsing it to store only the access
+    token in a separate file so we can hand that to the Hack-For-A-
+    Cause teams. The file name for the tokens would be
+    access_token_from_config_<team_name>.txt and is placed in the
+    current directory with the other yml files created.
+    """
+    with open(config_file, "r") as file_contents:
+        dict_of_contents = yaml.load(file_contents)
+
+    filename_for_token = "access_token_from_config_"+team_name+".txt"
+
+    access_token = (dict_of_contents['users'][0]['user']['auth-provider']
```

Comment:
I think this is just pulling out the access token for the 1st user in the entire cluster which is me. What we need to do is create a new user for the team and get the kubeconfig file for that user.

There's several ways to do this: https://cloud.google.com/iam/docs/overview. Scott and I are meeting today and will discuss options. In any event this needs to do the following:

1. Add a user to the cluster: https://cloud.google.com/kubernetes-engine/docs/how-to/iam. This will be done via `gcloud` commands
2. Add RBAC policies to limit that user to just their namespace: https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control. This will be done via `kubectl` commands.
3. Get a kubeconfig file for the team probably via `gcloud container clusters get-credentials ` (unless we go with service accounts)

This is all a bit complex so I might take this PR and add that stuff to it later today.